src/clr-angular/data/datagrid/datagrid-column-toggle.ts

```diff
@@ -28,7 +28,7 @@ import { DatagridColumnChanges } from './enums/column-changes.enum';
     <div class="column-switch"
          *clrPopoverOld="open; anchor: anchor; anchorPoint: anchorPoint; popoverPoint: popoverPoint">
       <div class="switch-header">
-        <ng-container *ngIf="!customToggleTitle">Show Columns</ng-container>
+        <ng-container *ngIf="!customToggleTitle">{{commonStrings.showColumns}}</ng-container>
```

Comment:
Now they really don't need to have `*ngIf="!customToggleTitle"` right? because they don't need to project their custom content in order to provide non-english text.

Reply:
I'm just thinking outloud here but, what if a consumer needs/wants still need to override the common string, just for one datagrid in the app but not others?

Reply:
@Shijir We need the ngIf for backwards compat for the deprecation. 
@hippee-lee I've never heard that ask, but have heard someone want to override the string once instead of in every datagrid like this old pattern requires.